vNext/extensions/applicationinsights-analytics-js/src/JavaScriptSDK/ApplicationInsights.ts

```diff
@@ -26,7 +26,7 @@ const durationProperty: string = "duration";
 export class ApplicationInsights implements IAppInsights, ITelemetryPlugin, IAppInsightsInternal {
     public static Version = "2.0.1-beta";
     public initialize: (config: IConfiguration, core: IAppInsightsCore, extensions: IPlugin[]) => void;
-    public identifier: string = "ApplicationInsightsAnalytics";
+    public identifier: string = "ApplicationInsightsAnalytics"; // do not change name or priority
```

Comment:
Any particular reason the priority cannot be changed?

Reply:
when customers take dependency on our packages, priority changing can cause issues. Documentatioin for priority coming up today. No packages should update their priority unless there is a requirement

Reply:
Just wanted to understand if this plugin have any extra particular constraint

Reply:
Got it, none. Its the same constraints for all plugins. Once finalized, dont change willy nilly.